pkg/clients/page.go

```diff
@@ -1,8 +1,5 @@
 package clients
 
-// Metadata represents arbitrary JSON.
```

Comment:
Since the Groups page does not embed Page, you can move all the ClientsPage data here and use only this one.